pkg/provider/cloud/kubevirt/csi.go

```diff
@@ -147,3 +150,67 @@ func reconcileInfraTokenAccess(ctx context.Context, namespace string, client ctr
 
 	return nil
 }
+
+func DataVolumeClonerClusterRoleCreator(name string) reconciling.NamedClusterRoleCreatorGetter {
```

Comment:
Should this be in `csi.go` or `cdi.go` (or `data_volume.go`)?

Reply:
I will move this to `data_volume.go`.